### DISCUSSION TITLE
Виджет CDEK и cargo.set

### DISCUSSION THREAD

**Discussion by FAUSTddd** [cdek-it/widget#84] [Ideas]
Здравствуйте. Планируется ли когда-то добавить объекту cargo метод set? Вроде вышла 3 версия а со времен 2 версии так и не добавили. 
При отправке cargo.reset и дополнительные cargo.add допустим у меня 100 товаров, каждый раз когда идет отправка reset и add идет отправка на сервер. соответственно 101 запросов. а при set был бы 1. Что я и добавил в 2 версии. но на 3 перейти не могу. В 4 добавите может?))

**Comment by vermorag** [comment]
Добрый день. 
Вопрос только в сокращении количества запросов на бэк, или в том, чтобы не вызывать `add` в цикле, а использовать только один раз `set`?

**Comment by FAUSTddd** [comment]
Да и в этом тоже. А так же иногда ответ расчета приходит раньше чем предыдущий, т.к. много запросов. Да и вообще почему не добавить? Создал массив, закинул, получил значение, всё. Многим облегчило бы жизнь
